lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app-helm/helm/lighty-rnc-app-helm/values.yaml

```diff
@@ -69,6 +69,14 @@ ingress:
   exposeManagement: false
   managementHost: "management.lighty.io"
 
+deploymentSecurity:
+  usePodSecurity: true
```

Comment:
Do we have use case when we do not want use "security"?

Reply:
Removed optional security for deployment